package/yapsy/PluginManager.py

```diff
@@ -442,16 +442,16 @@ def locatePlugins(self):
 		"""
 		self._candidates, npc = self.getPluginLocator().locatePlugins()
 	
-	def loadPlugins(self, callback=None):
+	def loadPlugins(self, callback=None, callback_after=None):
```

Comment:
For information: adding an additional parameter to loadPlugins requires (for consistency) that this same parameter be added to other plugin manager.

This is mainly because of a design flaw in yapsy (my bad) and I'll fix that after merging your commit.